cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -76,6 +78,39 @@ def new_snapshot(test_vm, has_name=True, memory=False):
 
 @pytest.mark.uncollectif(lambda provider:
     not provider.one_of(RHEVMProvider, VMwareProvider) or
+    (provider.one_of(RHEVMProvider) and provider.version < 4) or
+    current_version() < '5.8')
+def test_memory_checkbox(small_test_vm, provider, soft_assert):
+    # Make sure the VM is powered on
+    small_test_vm.power_control_from_cfme(option=small_test_vm.POWER_ON, cancel=False)
+    # Try to create snapshot with memory on powered on VM
+    if provider.one_of(RHEVMProvider):
+        snapshot1 = new_snapshot(small_test_vm, has_name=False, memory=True)
```

Comment:
**optional** could `has_name=not provider.one_of(RHEVMProvider)` be used ?

Reply:
Yes, good idea.